### DISCUSSION TITLE
Is there an option to set a collection as a Public and the customer doesn't need to login to see the information?

### DISCUSSION THREAD

**Discussion by krasho** [saleor/saleor#14511] [Deployment]
Hello

I'm trying to set some collections as public to show the products in my homepage when the users aren't logged yet. is there an option in the dashboard or the graphql to do that?

thanks

**Comment by timuric** [comment]
Hi @krasho all collections are public; make sure that products that they include are listed in channels you query

**Comment by krasho** [comment]
Hello Timuric 

The collection wasn't available

thanks
